doc/api/perf_hooks.md

```diff
@@ -98,6 +98,27 @@ added: v8.5.0
 Returns the current high resolution millisecond timestamp, where 0 represents
 the start of the current `node` process.
 
+### performance.startEventLoopTimer()
+<!-- YAML
+added: REPLACEME
+-->
+
+* Returns: {boolean} True if the timer was started. False if the timer has
+           already been started.
+
+Starts monitoring the event loop for delays. The accumulated event loop delay
+is reported using `performance.nodeTiming.eventLoopDelay`.
```

Comment:
Nit: `using` seems awkward here. `in`, `by`, or `as` all seem better to me.

Reply:
Feel free to make an edit suggestion and I'll accept it

Reply:
```suggestion
is reported as `performance.nodeTiming.eventLoopDelay`.
```

Feel free to pick a different preposition if that one doesn't seem quite right.